.swiftlint.yml

```diff
@@ -0,0 +1,63 @@
+excluded:
+  - Pods
+
+file_length:
+  - 800
```

Comment:
I'd like to lower these, thoughts? (maybe see if the default values don't through any flags?)

Reply:
Hmm, yeah i think we can lower it, thoughts on a better number?